config/opensearch_dashboards.yml

```diff
@@ -371,7 +371,7 @@
 # dynamic_config_service.enabled: false
 
 # Set the value to true to enable direct data import from a file
-# data_importer.enabled: false
+data_importer.enabled: true
```

Comment:
i'd recommend not source controlling this config file. historically, people have created PRs with passwords or IPs that they had to clean up later.

You can add a new package.json script or you can start OSD like

```
yarn start --data_importer.enabled=true --opensearch_security.enabled=false
```

Reply:
+1, I usually use git skip. But TIL you can add configs on the fly